smalld/smalld.py

```diff
@@ -97,9 +107,6 @@ def send_gateway_payload(self, data):
         logger.debug("gateway payload sent: %s", payload)
         self.gateway.send(payload)
 
-    def reconnect(self):
```

Comment:
Ah, sorry I think my last coment was unclear.

This reconnect method will be needed. This is, for example, how a listener can trigger a reconnect if needed.